src/ar.erl

```diff
@@ -436,13 +439,13 @@ start(
 		{http_bridge_node, Bridge}
 	]),
 	ar_randomx_state:start_block_polling(),
-	case Polling of
+	PollingArgs = [{trusted_peers, Peers}] ++ case Polling of
 		true ->
-			ar_meta_db:put(polling_mode, true),
-			ar_poller:start(Node, Peers);
+			[{polling_interval, 10 * 1000}];
 		false ->
-			do_nothing
+			[]
 	end,
+	{ok, _} = ar_poller_sup:start_link(PollingArgs),
```

Comment:
How come we always start `ar_poller`? Especially nice it seems that it'll start polling anyway